development/java/Rubberduck.Parsing/Grammar/build.gradle

```diff
@@ -0,0 +1,52 @@
+buildscript {
+    repositories {
+        mavenCentral();
+    }
+
+    dependencies {
+        classpath group: 'org.antlr', name: 'antlr4', version: '4.7.2'
+    }
+}
+
+plugins {
+    id 'java'
+}
+
+group 'com.rubberduckvba.rubberduck.parsing'
+version '1'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile group: 'org.antlr', name: 'antlr4-runtime', version: '4.7.2'
+}
+
+def grammerCodeGenDest = "${projectDir}/src/main/gen"
+def grammarSource = "${projectDir}/../../../../Rubberduck.Parsing/Grammar/"
+def grammarDest = "${projectDir}/src/main/antlr/com/rubberduckvba/rubberduck/parsing/grammar"
+
+sourceSets {
+    main {
+        java {
+            srcDir "${grammerCodeGenDest}"
+        }
+    }
+}
+
+task copyGrammarFiles(type: Copy) {
+    from grammarSource
+    into grammarDest
```

Comment:
~~I think we'll want the files that are copied over by this task to not be versioned. Can we explicitly add them to the gitignore?~~

I see that all .g4 files are ignored in a `.gitignore` scoped to the target directory, so that's pretty cool :)